bitnami/gitea/README.md

```diff
@@ -0,0 +1,358 @@
+<!--- app-name: Gitea -->
+
+# Gitea packaged by Bitnami
+
+Gitea is a lightweight code hosting solution. Written in Go, features low resource consumption, easy upgrades and multiple databases.
+
+[Overview of Gitea](https://gitea.org/)
```

Comment:
Looks like the wrong URL, it redirects to spam. I think the correct one is https://gitea.io.